src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -136,14 +136,32 @@ export function start(manifest) {
 
 The following methods are provided:
 
-##### `app.render(request)`
+##### `app.render(request, routeData, locals)`
 
 This method calls the Astro page that matches the request, renders it, and returns a Promise to a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) object. This also works for API routes, that do not render pages.
 
 ```js
 const response = await app.render(request);
 ```
 
+The method accepts a mandatory `request` arguments, and two other optionals arguments: `routeData` and `locals`.
+
+[//]: # (TODO) document `routeData`
```

Comment:
```suggestion
The method accepts a mandatory `request` argument, and two other optionals arguments: [`routeData`](en/reference/integrations-reference/#routedata-type-reference) and [`locals`](https://docs.astro.build/en/guides/middleware/#locals).
```
We could start with just a link like this, then figure out what we want to say about `routeData`?

With a link to `locals`, we could also change the sentence below to explain what's happening in the example itself, since the explanation for `locals` is provided elsewhere.

Reply:
> hen figure out what we want to say about `routeData`?

If that's fine with you, it's fine with me :)